test-suite/js/src/test/require-modules/org/scalajs/testsuite/jsinterop/ModulesTest.scala

```diff
@@ -124,71 +108,67 @@ class ModulesTest {
 
 package object modulestestpackageobject {
   @js.native
-  @JSImport("querystring", "stringify")
-  def stringify(obj: js.Dictionary[String], sep: String = "&",
-      eq: String = "="): String = js.native
+  @JSImport(ModulesTest.modulePath, "ssum")
+  def ssum(x: Int, y: Int = 0): Int = js.native
```

Comment:
It's a bit confusing that the default value here is `0` but actually the .js file uses `1` as default value.